Tests/test_imagefont.py

```diff
@@ -121,6 +121,7 @@ def test_textsize_equal(self):
             size = draw.textsize(txt, ttf)
             draw.text((10, 10), txt, font=ttf)
             draw.rectangle((10, 10, 10 + size[0], 10 + size[1]))
+            del draw
```

Comment:
My understanding is that `del draw` performs a cleanup operation to save memory. It was present on some tests, but not others. If you'd like to go the other way, and remove it in order to reduce unnecessary lines of code in the test suite, just say the word.